servers/jax-rs/build.gradle.kts

```diff
@@ -81,6 +81,7 @@ dependencies {
   api("org.glassfish.jersey.test-framework.providers:jersey-test-framework-provider-jetty")
 
   api("org.jboss.weld.se:weld-se-core")
+  api(libs.hibernate.validator.cdi)
```

Comment:
Can you move to line 57? It fits better in the first block.

Reply:
fixed